ReactJS/Menu/Getting-Started.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started 
+title: Getting Started with React Menu Bar | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential React Menu Bar component, it's elements and more details. 
 platform: React JS
 control: Menu
 documentation: ug
 keywords: ejmenu, menu, js menu
 ---
 
-# Getting Started 
+# Getting Started with React Menu Bar
 
 This section explains briefly about how to create a **Menu** control in your application with **JavaScript**. The **Essential JavaScript** **Menu** supports displaying a **Menu** of list-out items. This **Menu** is based on ul-li hierarchy, where the sub-list items are rendered as the sub-menu items. The **Menu** control can also be rendered with local and remote data source.  From the following guidelines, you can learn how to customize the **Menu** control for a website. In this case, **Syncfusion's** website **Menu** is discussed. The following screenshot displays the appearance of **Menu**.
 
-![](Getting-Started_images/Getting-Started_img1.png) 
+![Getting Started with React Menu Bar.](Getting-Started_images/react-menu-bar-getting-started.png) 
```

Comment:
There are some duplicates in alt text, so need to check.

Reply:
resolved